cmd/world/cardinal/create.go

```diff
@@ -17,6 +20,10 @@ const TemplateGitUrl = "https://github.com/Argus-Labs/starter-game-template.git"
 // Cobra Setup //
 /////////////////
 
+func init() {
+	createCmd.Flags().Bool(flagDebug, false, "Run in debug mode")
```

Comment:
I think it would be cleaner to make a method in logger that takes in a &combra.Command to attach this log flag to the command:

e.g.:
```
package logger
func AddLogFlag(cmd *cobra.Command) {
  cmd.Flags().Bool(flagDebug, false, "Run in debug mode")
}
```

Then you can use that helper in each command. This matches the pattern of logger.SetDebugMode(cmd) as well. It also means flagDebug only need to be declared in 1 place.

Reply:
Updated